runtime/doc/starting.txt

```diff
@@ -110,7 +110,10 @@ argument.
 		This can be used to find out where time is spent while loading
 		your |config|, plugins and opening the first file.
 		When {fname} already exists new messages are appended.
-
+		If your profiler script needs to know when to quit, use:
+>
+		nvim --startuptime nvim.log +'set ut=0 | au CursorHold * qa!'
```

Comment:
What if one changes `'updatetime'` on `vim.schedule()`

Reply:
`vim.schedule(function() vim.cmd('qa!') end)`? Doesn't work, it's executed before reaching `os_inchar`

Reply:
What?

Reply:
I don't really understand what would be the point of that. My intention behind `:set ut=0` was to make `CursorHold` trigger as soon as possible, and therefore quit neovim as soon as possible.

Reply:
So it doesn't work if someone has this kind of thing in their config.

Reply:
This is rare though.

Reply:
So to be clear, by "doesn't work" I mean that neovim quits before `--- NVIM IDLE ---` message.

IIRC `+`/`-c` commands are executed after the config, so I believe this should overwrite it. In worst case scenario if the user still manages to change `'updatetime'` to a higher value from the config, it will just take a little bit longer for neovim to quit.